quint/mempool/Mempool.qnt

```diff
@@ -0,0 +1,172 @@
+// -*- mode: Bluespec; -*-
+// A mempool is a replicated set of transactions which is used as an input by a ledger.
+// Below, we follow the specification given here:
+// https://github.com/cometbft/knowledge-base/blob/main/protocols/mempool-overview.md
+
+module ABCI {
+
+    import System.* from "System"
+    import Ledger.* from "Ledger"
+
+    // a transaction is committed at most once
+    pure def isValid(l: LedgerState, p: Process, t: Tx): bool = {
+      not(isCommittedFor(l, p, t))
```

Comment:
Indeed. In fact, it would be possible to have a more involved application, e.g., a key-value store (as in the actual test code base of the mempool module). I wanted some feedback before possibly doing this.